README.md

```diff
@@ -108,7 +108,26 @@ npm run beforeIntTest
 
 ## Deploy
 
-To deploy to the `dev` environment (also called `build`), commit to the `main` branch
+To deploy to the `dev` environment on the main stack `di-btm`, commit to the `main` branch, this will also deploy into
+the `build` environment in the build account.
+
+To deploy to your own on-demand stack:
+
+Log in to the `dev` account using your method of choice (`gds-cli`, AWS profiles, etc.).
```

Comment:
I think that, because of the information in the new starter guide, we log in to dev using the gds-cli like this: gds aws di-btm-dev -s
Do we replace dev with build or prod to log into the other environments?